idaes/models_extra/power_generation/costing/costing_dictionaries.py

```diff
@@ -64,3 +64,9 @@ def load_sCO2_costing_dictionary():
     with open(os.path.join(directory, "sCO2_costing_parameters.json"), "r") as file:
         sCO2_costing_params = json.load(file)
     return sCO2_costing_params
+
+
+def load_REE_costing_dictionary():
```

Comment:
Is this the right place for this? It seems really odd that something to do with REEs is ending up in the power plant sub-folder. This might indicate that the `power_generation/costing` folder really needs to be moved out to a `models_extra/costing` as it has spread beyond just power generation.

Reply:
I think it really should be moved before this is merged - otherwise it is likely to linger. Seeing as this is our public repo, we should make an effort to ensure that it is user friendly.